tools/python/boutiques/schema/examples/docopt_to_argparse/sample_pattern.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Unless we use these in tests (to verify the "expected" interpretation of command args), we should remove the `pattern` files from the repo